docs/guides/building-a-minecraft-demo.md

```diff
@@ -382,26 +382,28 @@ teleport to the end of the arc. Before, we wrote our own A-Frame components.
 But we can also use open source components already made from the community
 and just use them straight from HTML!
 
-[blink-controls]: https://github.com/fernandojsg/aframe-teleport-controls/
-For teleportation, there's a component called [blink-controls][blink-controls].
-Following the README, we add the component via a `<script>` tag and just set
-the `blink-controls` component on the controller on the entity:
+Even though we haven't explicity defined one, our scene has an implicit camera 
+component from which the user views the scene. Since teleporting is going to be
+moving this camera along with the player, we'll need to make it explicit and 
+part of the player:
 
 ```html
-<script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
-<script src="https://cdn.jsdelivr.net/npm/aframe-blink-controls/dist/aframe-blink-controls.min.js"></script>
-
-<!-- ... -->
-
-<a-entity id="teleHand" hand-controls="hand: left" teleport-controls></a-entity>
-<a-entity id="blockHand" hand-controls="hand: right"></a-entity>
+<a-entity id="player">
```

Comment:
Removing aframe script intentional?

Reply:
Yes, because I wanted to only show the code that was relevant to the section. I hadn't realized the code blocks were made to be executable on their own!